### PR TITLE
test_CCTRI-1720

### PR DESCRIPTION
**Pull request by mistergreen22**

### CODE REVIEW

File: tests/functional/tests/test_ctr_05_api.py [hunk 1]
```diff
@@ -456,3 +461,115 @@ def test_python_module_negative_token(token, error):
...
+    user = response['user']
+    org = response['org']
+
+    assert user['role'] == 'admin'
+    assert user['scopes']
+    assert user['updated-at']
+    assert user['user-email'] == 'olshtaie+qasandbox@cisco.com'
+    assert user['org-id']
+    assert user['user-id']
+    assert user['enabled?'] is True
```
Comment (GeorgiyII) [inline tests/functional/tests/test_ctr_05_api.py:490]: It works without `is True`

File: tests/functional/tests/test_ctr_05_api.py [hunk 2]
```diff
@@ -456,3 +461,115 @@ def test_python_module_negative_token(token, error):
...
+    assert user['user-nick'] == 'ATQC account'
+    assert user['idp-mappings'][0]['idp']
+    assert user['idp-mappings'][0]['user-identity-id']
+    assert user['idp-mappings'][0]['organization-id']
+    assert user['idp-mappings'][0]['enabled?'] is True
+    assert user['last-logged-at']
+
+    assert org['updated-at']
+    assert org['name'] == 'cisco'
+    assert org['allow-all-role-to-login'] is False
```
Comment (GeorgiyII) [inline tests/functional/tests/test_ctr_05_api.py:501]: it works `assert not org['allow-all-role-to-login']` without` is False

File: tests/functional/tests/test_ctr_05_api.py [hunk 3]
```diff
@@ -456,3 +461,115 @@ def test_python_module_negative_token(token, error):
...
+    assert org['enabled?'] is True
```
Comment (GeorgiyII) [inline tests/functional/tests/test_ctr_05_api.py:502]: It works without `is True`

File: tests/functional/tests/test_ctr_05_api.py [hunk 4]
```diff
@@ -456,3 +461,115 @@ def test_python_module_negative_token(token, error):
...
+    assert org['activation-status'] == 'activated'
+    assert org['scim-status'] == 'activated'
+    assert org['id']
+    assert org['created-at']
+    assert org['allow-all-role-to-login-editable?'] is True
```
Comment (GeorgiyII) [inline tests/functional/tests/test_ctr_05_api.py:507]: It works without `is True`

File: tests/functional/tests/test_ctr_05_api.py [hunk 5]
```diff
@@ -456,3 +461,115 @@ def test_python_module_negative_token(token, error):
...
+    Expectedresults: The response body contains all needed data
+
+    Importance: Critical
+    """
+    response = get_org(**{'headers': module_headers})
+
+    assert response['updated-at']
+    assert response['name'] == 'cisco'
+    assert response['allow-all-role-to-login'] is False
+    assert response['enabled?'] is True
```
Comment (GeorgiyII) [inline tests/functional/tests/test_ctr_05_api.py:528]: the same

**Review by GeorgiyII — APPROVED** [review]

**Review by AlinaBlahun — APPROVED** [review]

**Pull request merged**